actix-http/src/ws/codec.rs

```diff
@@ -68,6 +86,27 @@ impl Codec {
         self.server = false;
         self
     }
+
+    fn combine_payload(&mut self, payload: Option<BytesMut>) -> Option<BytesMut> {
```

Comment:
i think it would be better to bubble up frames with continuation to application layer, application can decide if it wants to accumulate payload or do something with partial payload